src/index.js

```diff
@@ -55,6 +55,7 @@ nativeBindings.nativeWindow.setEventLoop(eventLoopNative);
 
 const GlobalContext = require('./GlobalContext');
 GlobalContext.commands = [];
+GlobalContext.exokit = core;
```

Comment:
Instead of globally exposing, is there a way we could require the core instead?

Reply:
Note that attempting to require the core from within webworkers seems to explode. It may be a better idea to expose this globally. Not sure yet.